org.eclipse.buildship.core/src/main/java/org/eclipse/buildship/core/internal/workspace/GradleClasspathContainerRuntimeClasspathEntryResolver.java

```diff
@@ -161,6 +163,9 @@ private void collectContainerRuntimeClasspathWithGradleScopes(IClasspathContaine
             throw new GradlePluginsRuntimeException("JavaRuntime.resolveRuntimeClasspathEntry() should not be called when Buildship is installed for Eclipse 4.8", e);
         }
     }
+    private boolean hasModuleAttribute(IClasspathEntry entry) {
+        return false && hasEnabledBooleanAttribute(IClasspathAttribute.MODULE, entry);
```

Comment:
```suggestion
        return hasEnabledBooleanAttribute(IClasspathAttribute.MODULE, entry);
```

Reply:
Oh, that is silly! I had wanted to comment it out and see if it really made the difference, or whether it was already fixed. Forgot to comment it in again ... :-(